### PR TITLE
Chapter 14 review

### PR DESCRIPTION
**Pull request by marquicodes**
These are the changes suggested after reviewing Chapter 14.

- Fixes queries to update nickname where the column name was wrong (nickname instead of nick),
- Updates `changeNickNameWithInput` mutation to return only the ID as defined in `gql-schema.js`,
- Removes the reference to person dataloader in `app-without-dataloaders.js` file,
- Renames table names to all lowercase as it is in the book

### CODE REVIEW

File: Chapter 14/data-loaders/friend.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const DataLoader = require('dataloader')
 module.exports = function buildDataLoader (app) {
   const dl = new DataLoader(async function fetcher (ids) {
     const secureIds = ids.map(id => SQL`${id}`)
-    const sql = SQL`SELECT * FROM Friend WHERE personId IN (${SQL.glue(secureIds, ',')})`
+    const sql = SQL`SELECT * FROM friend WHERE personId IN (${SQL.glue(secureIds, ',')})`
```
Comment (Eomm) [inline Chapter 14/data-loaders/friend.js:9]: I read your feedback here, but I fixed all the occurrences in `Pascal` case to be homogeneous - I must say that I don't have strong opinions on this, just habit, and I know it is a bikeshedding topic

Comment (marquicodes) [reply Chapter 14/data-loaders/friend.js:9]: I am used to table names in lowercase, the same case was used in the queries in the book before Dataloaders introduction. That's why I made this change.

File: Chapter 14/app.js [hunk 2]
```diff
@@ -41,14 +41,14 @@ async function run () {
...
-        const sql = SQL`UPDATE person SET nickName = ${input.nick} WHERE id = ${input.personId}`
+        const sql = SQL`UPDATE person SET nick = ${input.nick} WHERE id = ${input.personId}`
         const { changes } = await context.app.sqlite.run(sql)
         if (changes === 0) {
           throw new Error(`Person id ${input.personId} not found`)
         }
         const person = await context.personDL.load(input.personId)
         context.reply.log.debug({ person }, 'Read updated person')
-        return person
+        return person.id
```
[the remark this answers is not in the document]
Comment (marquicodes) [reply Chapter 14/app.js:51]: I didn't know that you decided to change the return type. I didn't get any update and the code as it is didn't work.

[the remark this answers is not in the document]
Comment (Eomm) [reply Chapter 14/app.js:51]: Here the changes, I will add some tests to be sure 👍🏽

**Comment by Eomm** [comment]
Supersedes by #27

**Pull request closed without merging**